### ISSUE TITLE
Tips to accelarate

### ISSUE THREAD

**Issue by eoteroe** [lejon/TSne.jl#36]
Wondering if is possible to use threads. I was applying tsne to a dataset of size(50K,10) and it is really slow. 
Can you give advice to handle big datasets?

**Comment by lejon** [comment]
Yes, unfortunately it is very slow. The current implementation is the original version of t-SNE and there is not much you can do there in terms of threads unfortunately. I have been working on a version of Barnes Hut t-SNE which is both much faster and parallelizable but I have not gotten it to work well yet unfortunately.As far as I know, if you need a fast version on large datasets you unfortunately have to go to another language. I have a Java version in another GitHub repo that is very fast, but there are other fast ones also. On 20 Jun 2024, at 22:20, Eliezer Otero Espinoza ***@***.***> wrote:﻿
Wondering if is possible to use threads. I was applying tsne to a dataset of size(50K,10) and it is really slow.
Can you give advice to handle big datasets?
